shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/response/GlobalResponse.kt

```diff
@@ -83,4 +84,15 @@ data class GlobalResponse(
             }
             .distinct()
     }
+
+    fun getAlertAffectedStops(alert: Alert?, routes: List<Route>?): List<Stop>? {
```

Comment:
suggestion: I know this wasn't tested specifically in either platform, but would be nice to have a unit test or two for it now that it is pulled out!

Reply:
Added.